docs/usage/configuration-options.md

```diff
@@ -196,20 +196,37 @@ The only platform that supports `automergeStrategy` is Bitbucket Cloud.
 
 This setting is only applicable if you opt in to configure `automerge` to `true` for any of your dependencies.
 
-Automerging defaults to using Pull Requests (`automergeType="pr"`).
+### pr
+
+Automerging defaults to merging Pull Requests by Renovate itself.
+
 In that case Renovate first creates a branch and associated Pull Request, and then automerges the PR on a subsequent run once it detects the PR's status checks are "green".
 If by the next run the PR is already behind the base branch it will be automatically rebased, because Renovate only automerges branches which are up-to-date and green.
 If Renovate is scheduled for hourly runs on the repository but commits are made every 15 minutes to the main branch, then an automerge like this will keep getting deferred with every rebase.
 
 Note: if you have no tests but still want Renovate to automerge, you need to add `"ignoreTests": true` to your configuration.
 
+### platform-automerge
+
+With `platform-automerge`, you can speed up the automerge process by using platform automerge facilities. Setting this will configure PRs to be merged after all (if any) branch policies have been met. Currently available for Azure and GitLab.
```

Comment:
```suggestion
With `platform-automerge`, you can speed up the automerge process by using platform automerge facilities.
Setting this will configure PRs to be merged after all (if any) branch policies have been met.
Currently available for Azure and GitLab.
```

Let's use one sentence per line here. 😉